### DISCUSSION TITLE
Need Configuration Advice

### DISCUSSION THREAD

**Discussion by goparklubanic** [HaylLtd/cordova-plugin-background-geolocation#175] [Q&A]
Hi all. I am Nugie from a local IT bussines in Indonesia. Currently we are on experiment developing delivery tracking system. We use this plugin in Apache Cordova as mobile app / front end, and Laravel framework in back end.

I would like to consult about our  configuration. Here is some of them:
```
        url: 'https://ourbackend.com/api/set',
        syncUrl: 'https://ourbackend.com/api/set',
        httpHeaders:{
            'Content-type':'application/json; charset=UTF-8',
        },
        postTemplate:{
            lat: '@latitude',
            lon: '@longitude',
            dev: device.uuid,
        }
```
![access log](https://github.com/HaylLtd/cordova-background-geolocation-plugin/assets/1618589/ce45ea51-f1b0-4203-a8ec-3378221469fd)

The data were sent to the back end, but not recorded in database. We are so appreciate for any advice or suggestion.

**Comment by risinghero** [comment]
If you mean plugin sqlite database, then data should be deleted on successful receival by your website.

**Reply by goparklubanic** [reply]
What is curl or fetch syntax which similar or equal with this configuration?
url: 'https://ourbackend.com/api/set',
        syncUrl: 'https://ourbackend.com/api/set',
        httpHeaders:{
            'Content-type':'application/json; charset=UTF-8',
        },
        postTemplate:{
            lat: '@latitude',
            lon: '@longitude',
            dev: device.uuid,
        }
